<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item>
     <widget class="QGroupBox" name="backendGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string>Backend</string>
      </property>
      <widget class="QComboBox" name="backendComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>24</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>System Debug Infrastructure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Simulation TCP Interface</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="optionsGroup">
      <property name="title">
       <string>Options</string>
      </property>
      <widget class="QStackedWidget" name="optionsStack">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>19</y>
         <width>381</width>
         <height>191</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="dbgnocOptionsWidget">
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>381</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="dbgnocOptionsLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="dbgnocConnectionLabel">
              <property name="text">
               <string>Connection</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="dbgnocConnectionComboBox">
              <item>
               <property name="text">
                <string>USB</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>TCP</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="dbgnocConnectionsStack">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="dbgnocConnectionUSBWidget">
             <widget class="QWidget" name="formLayoutWidget_2">
              <property name="geometry">
               <rect>
                <x>-1</x>
                <y>-1</y>
                <width>381</width>
                <height>161</height>
               </rect>
              </property>
              <layout class="QFormLayout" name="dbgnocConnectionUSBLayout"/>
             </widget>
            </widget>
            <widget class="QWidget" name="dbgnocConnectionTCPWidget">
             <widget class="QWidget" name="formLayoutWidget_3">
              <property name="geometry">
               <rect>
                <x>-1</x>
                <y>-1</y>
                <width>381</width>
                <height>161</height>
               </rect>
              </property>
              <layout class="QFormLayout" name="dbgnocConnectionTCPLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="dbgnocTCPHostnameLabel">
                 <property name="text">
                  <string>Hostname</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="dbgnocTCPHostnameLineEdit">
                 <property name="text">
                  <string>localhost</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="dbgnocTCPPortLabel">
                 <property name="text">
                  <string>Port</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="dbgnocTCPPortSpinBox">
                 <property name="maximum">
                  <number>65535</number>
                 </property>
                 <property name="value">
                  <number>32000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="simtcpOptionsWidget">
        <widget class="QWidget" name="formLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>381</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="simtcpOptionsLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="simtcpHostnameLabel">
            <property name="text">
             <string>Hostname</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="simtcpHostnameLineEdit">
            <property name="text">
             <string>localhost</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="simtcpPortLabel">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="simtcpPortLineEdit">
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>22000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
